### PR TITLE
Processor delete

### LINKED ISSUE #205: Remove processors
When removing a processor it should confirm the action and should display a feedback on what is happening in the server side in a transparent way to keep users data protected.

### PR DESCRIPTION
**Pull request by fjcero**
Closes #205

### CODE REVIEW

File: admin/src/components/Processors/Processors.js [hunk 1]
```diff
@@ -8,17 +8,22 @@ const Processors = ({
   processors,
   onCreateProcessorClick,
   onEditProcessorClick,
+  onDeleteProcessorClick,
   isLoading,
   children
 }) => {
-  function onCreateProcessorHandler(e) {
+  function onCreateProcessorHandler() {
     onCreateProcessorClick && onCreateProcessorClick();
```
Comment (maksymilian-majer) [inline admin/src/components/Processors/Processors.js]: @Jakedavies had a suggestion to make the props required and not check if they exist – I fixed it in a later commit (see master)

File: admin/src/components/Processors/Processors.js [hunk 2]
```diff
@@ -8,17 +8,22 @@ const Processors = ({
...
   }
 
-  function onEditProcessorHandler(e, processor) {
+  function onEditProcessorHandler(processor) {
     onEditProcessorClick && onEditProcessorClick(processor);
```
Comment (maksymilian-majer) [inline admin/src/components/Processors/Processors.js]: here too

File: admin/src/components/Processors/Processors.js [hunk 3]
```diff
@@ -8,17 +8,22 @@ const Processors = ({
...
   }
 
+  function onDeleteProcessorHandler(processor) {
+    onDeleteProcessorClick && onDeleteProcessorClick(processor);
```
Comment (maksymilian-majer) [inline admin/src/components/Processors/Processors.js]: and here

File: admin/src/components/Processors/Processors.js (hunk 1 above)
Comment (fjcero) [reply admin/src/components/Processors/Processors.js]: 👍

**Pull request merged**